File: Services/UnionService/UnionService.Application/UseCases/CompanyUseCases/AddCompanyDateTimeChecker/AddCompanyDateTimeCheckerCommand.cs
using MediatR;
using UnionService.Domain.Enums;

namespace UnionService.Application.UseCases
{
    public record AddCompanyDateTimeCheckerCommand(
        string? username,
        int? companyId,
        DateTimeCheckerType? type,
        string? address,
        string? model
    ) : IRequest<Unit>;
}

File: FrontendApp/src/app/models/commands/unionservice/add-company-datetime-checker.command.ts
import { DateTimeCheckerType } from "../../../enums/datetime-checker-type.enum";

export interface AddCompanyDateTimeCheckerCommand {
  username: string | null;
  companyId: number | null;
  type: DateTimeCheckerType | null;
  address: string | null;
  model: string | null;
}
